packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditReducers.ts

```diff
@@ -157,57 +204,55 @@ export function applyFieldEdit(tree: FuzzView, fieldEdit: FieldEdit): void {
 
 function applySequenceFieldEdit(
 	tree: FuzzView,
-	field: FlexTreeSequenceField<readonly [Any]>,
+	parentNode: FuzzNode,
 	change: Insert | Remove | IntraFieldMove | CrossFieldMove,
 ): void {
 	switch (change.type) {
 		case "insert": {
-			field.editor.insert(
-				change.index,
-				mapTreeFieldFromCursor(cursorForJsonableTreeField(change.content)),
+			const insertValues = change.content.map((value) =>
+				generateFuzzLeafNode(value, tree.currentSchema),
 			);
+			for (let index = change.index; index < insertValues.length; index++) {
+				parentNode.sequenceChildren.insertAt(index, insertValues[index - change.index]);
+			}
+			Tree.parent(parentNode.sequenceChildren);
```

Comment:
remove?

Reply:
removed unnecessary call to `Tree.parent()`